### ISSUE TITLE
Drumkit editor?

### ISSUE THREAD

**Issue by mkruselj** [samlletas/syxg50-control-panel#15]
It's probably the last thing missing, I guess apart from supporting insertion vs system mode for Variation. 3.1, perhaps? 🙂

**Comment by samlletas** [comment]
Yeah I think drum setup is the last major feature missing, perhaps we can do a table similar to the overview page (with multiple tabs or even scrollable rows since we would need 72 of them).

Personally this is something I'll rarely use so I'm not sure when/if the feature will be added, but I agree that supporting drum setup would be nice 🙂

Regarding insertion/system mode, I wonder if supporting insertion mode is even worth the effort since the S-YXG50 can't use more than one variation type anyways (it doesn't support multiple insertion effects like some of the hardware MU models).

**Comment by mkruselj** [comment]
Yeah I would say having a menu for selecting which key to edit, and a MIDI learn mode to select the key to edit as you press the note, that would be awesome. And I agree, not many people will use this probably. But it's the completionist's way of doing things. I don't know how much of a completionist you are, though. 😛

**Comment by samlletas** [comment]
Your approach sounds better, specially the MIDI learn mode. I like the idea of the control panel supporting every major feature so only time will tell!
